pom.xml

```diff
@@ -26,6 +26,14 @@
     <tag>HEAD</tag>
   </scm>
 
+  <dependencies>
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>matrix-project</artifactId>
+          <version>1.4.1</version>
```

Comment:
Use 1.4 unless you are depending on a core version with the SECURITY-125 fix (so 1.596.1+).